doc/api/console.markdown

```diff
@@ -72,15 +72,15 @@ object. This is useful for inspecting large complicated objects. Defaults to
 - `colors` - if `true`, then the output will be styled with ANSI color codes.
 Defaults to `false`. Colors are customizable, see below.
 
-### console.time(timerName)
+### console.time(label)
```

Comment:
Thanks, revert filed in https://github.com/nodejs/node/pull/3590. I'm sorry for the trouble.